conf/routes

```diff
@@ -41,12 +41,12 @@ GET  /int/contribute                                controllers.Application.cont
 GET  /nz/contribute                                 controllers.Application.contributionsLanding(id="contributions-landing-page-nz")
 GET  /ca/contribute                                 controllers.Application.contributionsLanding(id="contributions-landing-page-ca")
 
-GET  /contribute/recurring                          controllers.RegularContributions.displayForm()
-GET  /contribute/recurring/thankyou                 controllers.RegularContributions.displayForm()
+GET  /contribute/recurring                          controllers.RegularContributions.displayForm(useNewSignIn: Option[Boolean])
```

Comment:
actually, I have an even better suggestion for the handling of this optional param: just make it `Boolean` and default it to `false`, right here in the routes file:

https://www.playframework.com/documentation/2.6.x/ScalaRouting#parameters-with-default-values

No need for `useNewSignIn.getOrElse(false)`